service/routes/docs.js

```diff
@@ -297,12 +299,10 @@ function route(req, res, next) {
 		.then(locals => {
 			template(locals.pageName)
 				.then(templFn => res.send(templFn(locals)))
-				.catch(function() { next(); })
+				.catch(() => next())
```

Comment:
I think if we do that, the error will cause express to bail.  If you pass an object to `next` it tells express to skip all remaining route middleware and jump to the first error middleware.